testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/MultinodeHiveCaching.java

```diff
@@ -72,7 +72,7 @@ public void extendEnvironment(Environment.Builder builder)
     {
```

Comment:
Why this class do not use `TrinoMultinode`?

Reply:
This has multiple workers... while `TrinoMultinode` has only one worker